src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user = new UserService();
```

Comment:
Do you really need to create `UserService()` on each call of `login` or should we change declaration level for this variable?

Reply:
No i don`t). I`ve fixed login method. Pls check it now.